src/dowel/csv_output.py

```diff
@@ -30,26 +32,41 @@ def record(self, data, prefix=''):
         if isinstance(data, TabularInput):
             to_csv = data.as_primitive_dict
 
-            if not to_csv.keys() and not self._writer:
+            # if not to_csv.keys() and not self._writer:
```

Comment:
It's always better to delete blocks of code instead of commenting them out. They're already in the git history if we need them again :)

Reply:
Got it. I'm used to commenting in my own projects...